clients/src/main/java/org/apache/kafka/clients/producer/internals/TransactionManager.java

```diff
@@ -848,6 +848,8 @@ public void handleResponse(AbstractResponse response) {
                         coordinatorReloaded = true;
                         lookupCoordinator(FindCoordinatorRequest.CoordinatorType.GROUP, builder.consumerGroupId());
                     }
+                } else if (error == Errors.UNKNOWN_TOPIC_OR_PARTITION) {
```

Comment:
Nvm, I thought it is a general response handler but it is only for `txnCommitOffset`.